python/ray/_raylet.pyx

```diff
@@ -932,7 +932,7 @@ cdef class CoreWorker:
         return data.get() == NULL
 
     def put_file_like_object(
-            self, metadata, data_size, file_like, ObjectRef object_ref=None):
+            self, metadata, data_size, file_like, ObjectRef object_ref):
```

Comment:
Appears to only be used for [restoring spilled objects](https://github.com/ray-project/ray/blob/4d830039927142659a17d3a4d9974cfdf31f9fe4/python/ray/external_storage.py#L91), in which case I think that an `object_ref` will always be provided.

Reply:
Thanks for the cleanup!